packages/user/src/components.tsx

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+import { DOMUtils } from '@jupyterlab/apputils';
+import { userIcon } from '@jupyterlab/ui-components';
+
+import * as React from 'react';
+
+import { User } from './model';
+import { IUser } from './tokens';
+import { getInitials } from './utils';
+
+/**
+ * React component to render the user icon.
+ *
+ * @param user User information as IUser or JSON object.
+ */
+export const getUserIcon = (user: IUser | IUser.User) => {
```

Comment:
Prefer that structured otherwise the react component will be anonymous.

```suggestion
export function getUserIcon(user: IUser | IUser.User): JSX.Element {
```